src/config/api/polkadot/connectApi.ts

```diff
@@ -160,6 +160,8 @@ export async function connectApi(
     fallbackConnection({ networkIdx, endpoint });
   }
 
+  // Memo: wait for the wallet extensions to load in Firefox
+  await wait(400);
```

Comment:
We can't be sure that extension will be loaded in 400ms. Is there a event we can listen or similar?

Reply:
You are right. How about [this](https://github.com/AstarNetwork/astar-apps/pull/386/commits/8e43b9f68f04079709c552763b31b92a362c0146#diff-f3c7b24fa7deac4c9c27bae263642406de5d5e7e4abc997af4afae4f24e04109R17) solution?